clients/cli/src/main/java/org/apache/gravitino/cli/commands/MetalakeDetails.java

```diff
@@ -54,6 +58,10 @@ public void handle() {
       return;
     }
 
-    System.out.println(metalake + "," + comment);
+    List<String> headers = Arrays.asList("metalake", "comment");
+    List<List<String>> rows = new ArrayList<>();
+    rows.add(Arrays.asList(metalake, comment));
+    TablePrinter tablePrinter = new TablePrinter();
+    tablePrinter.print(headers, rows);
```

Comment:
Same as above.

A core principle of Unix CLI design is that commands should be both human and machine-readable, producing clean output that can easily be piped into other scripts. As the Basics of Unix Philosophy states:"Expect the output of every program to become the input to another, as yet unknown, program. Don't clutter output with extraneous information."

If a user had written some scripts, these changes would break those scripts; you need to carefully consider any changes to the command output or, better still, in this case, add these changes under a flag so the existing output is not changed.

Reply:
Thank you for pointing this out. I'll update the changes to be optional via a flag.